charts/backbeat/templates/gc/deployment.yaml

```diff
@@ -40,10 +40,12 @@ spec:
             httpGet:
               path: {{ .Values.health.path.liveness}}
               port: {{ .Values.health.port }}
+            initialDelaySeconds: 30
```

Comment:
I think the `periodDelay` of something like 10-15 would be very useful in this PR as well since the dependency services (kafka and zookeeper) usually take the longest to come up. This way the exponential `CrashLoopBackoff` doesn't cause the pods to quickly build up a long retry period.

Reply:
Since the problem in this PR does not seem related to this initial delay but to conflicts with other pending PRs, let's keep those useful hints aside for a follow-up PR.